docs/data/charts/funnel/FunnelCurves.js

```diff
@@ -57,6 +65,7 @@ export default function FunnelCurves() {
 <FunnelChart
   series={[{ curve: '${props.curveType}' }]}
   gap={${props.gap}}
+  ${props.curveType === 'bump' ? '// ' : ''}borderRadius={${props.borderRadius}}
```

Comment:
We are we commenting the border radius in this case?

Reply:
Yes, it doesn't work with the bump curve.

Reply:
>Should we not write the prop at all, then?

I tried disabling but there is no api for that in this type of component/test. I felt commenting it out would be more clear that it doesn't work, though it is up to discussion 😄